api/nakadi-event-bus-api.yaml

```diff
@@ -1217,6 +1221,7 @@ definitions:
       - name
       - category
       - owning_application
+      - enrichment_strategies
```

Comment:
Then I believe it should not be in a list of required fields.
I think it's ok - we can just say that for certain categories we have additional event-type validation.